config/orchestrations/admission-webhook/0.2/kabanero-operator-admission-webhook-config.yaml

```diff
@@ -117,3 +117,63 @@ webhooks:
     scope: '*'
   sideEffects: Unknown
   timeoutSeconds: 30
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: webhook.operator.kabanero.io
+webhooks:
+- admissionReviewVersions:
+  - v1beta1
+  clientConfig:
+    caBundle: {{ .caBundle }}
+    service:
+      name: kabanero-operator-admission-webhook
+      namespace: kabanero
+      path: /validate-stacks
+  failurePolicy: Fail
+  name: validating.stack.kabanero.io
+  namespaceSelector:
+    matchExpressions:
+    - key: control-plane
+      operator: DoesNotExist
+  rules:
+  - apiGroups:
+    - kabanero.io
+    apiVersions:
+    - v1alpha1
+    operations:
+    - CREATE
+    - UPDATE
+    resources:
+    - stacks
+    scope: '*'
+  sideEffects: Unknown
+  timeoutSeconds: 30
+- admissionReviewVersions:
```

Comment:
This one is not necessary - well I guess it depends on how you merge this validating webhook config with the previous one.